Microsoft.Maui.Core.props" could not be loaded. The process cannot access the file

### Description

Occasionally our release pipeline is failing with this

https://dev.azure.com/xamarin/public/_build/results?buildId=123137&view=logs&jobId=5239058e-6f59-5db5-6c0e-6f79eefcd049&j=5239058e-6f59-5db5-6c0e-6f79eefcd049&t=88825977-979d-51fd-4bb4-90e109dad17e

Ask @PureWeen (24615-build-windows.zip) for the binlogs if the above pipeline has been cleared

```
D:\a\_work\1\s\src\Maui.InTree.props(7,5): error MSB4024: The imported project file "D:\a\_work\1\s\.buildtasks\Microsoft.Maui.Core.props" could not be loaded. The process cannot access the file 'D:\a\_work\1\s\.buildtasks\Microsoft.Maui.Core.props' because it is being used by another process. [D:\a\_work\1\s\src\Compatibility\ControlGallery\src\iOS\Compatibility.ControlGallery.iOS.csproj]
```

--- a/src/Controls/src/BindingSourceGen/Controls.BindingSourceGen.csproj
+++ b/src/Controls/src/BindingSourceGen/Controls.BindingSourceGen.csproj
@@ -27,6 +27,6 @@
       <_CopyItems Include="$(TargetDir)*.dll" Exclude="$(TargetDir)System.*.dll" />
       <_CopyItems Include="$(TargetDir)*.pdb" Exclude="$(TargetDir)System.*.pdb" />
     </ItemGroup>
-    <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_MauiBuildTasksLocation)" ContinueOnError="true" Retries="0" />
+    <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_MauiBuildTasksLocation)" ContinueOnError="true" Retries="3" RetryDelayMilliseconds="1000" />
   </Target>
 </Project>

--- a/src/Controls/src/Build.Tasks/Controls.Build.Tasks.csproj
+++ b/src/Controls/src/Build.Tasks/Controls.Build.Tasks.csproj
@@ -73,7 +73,7 @@
 			<_CopyItems Include="$(TargetDir)*.dll" Exclude="$(TargetDir)System.*.dll;$(TargetDir)Microsoft.Build.*" />
 			<_CopyItems Include="$(TargetDir)*.pdb" Exclude="$(TargetDir)System.*.pdb;$(TargetDir)Microsoft.Build.*" />
 		</ItemGroup>
-		<Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_MauiBuildTasksLocation)%(RecursiveDir)" ContinueOnError="true" Retries="0" />
+		<Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_MauiBuildTasksLocation)%(RecursiveDir)" ContinueOnError="true" Retries="3" RetryDelayMilliseconds="1000" />
 	</Target>
 
 	<ItemGroup>

--- a/src/Controls/src/SourceGen/Controls.SourceGen.csproj
+++ b/src/Controls/src/SourceGen/Controls.SourceGen.csproj
@@ -63,7 +63,7 @@
       <_CopyItems Include="$(TargetDir)*.dll" Exclude="$(TargetDir)System.*.dll" />
       <_CopyItems Include="$(TargetDir)*.pdb" Exclude="$(TargetDir)System.*.pdb" />
     </ItemGroup>
-    <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_MauiBuildTasksLocation)" ContinueOnError="true" Retries="0" />
+    <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_MauiBuildTasksLocation)" ContinueOnError="true" Retries="3" RetryDelayMilliseconds="1000" />
   </Target>
 
 </Project>

--- a/src/Core/src/Core.csproj
+++ b/src/Core/src/Core.csproj
@@ -95,7 +95,7 @@
       <_CopyItems Include="nuget\buildTransitive\net-windows\**" SubFolder="net$(_MauiMinimumSupportedDotNetTfm)-windows$(MinimumWindowsTargetFrameworkVersion)" />
       <_CopyItems Include="$(IntermediateOutputPath)Microsoft.Maui.Core.BundledVersions.targets" SubFolder="" />
     </ItemGroup>
-    <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_MauiBuildTasksLocation)%(SubFolder)%(RecursiveDir)" ContinueOnError="true" Retries="0" />
+    <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_MauiBuildTasksLocation)%(SubFolder)%(RecursiveDir)" ContinueOnError="true" Retries="3" RetryDelayMilliseconds="1000" />
   </Target>
 
   <Import Project="$(MauiSrcDirectory)Workload\Shared\LibraryPacks.targets" />

--- a/src/SingleProject/Resizetizer/src/Resizetizer.csproj
+++ b/src/SingleProject/Resizetizer/src/Resizetizer.csproj
@@ -71,13 +71,13 @@
       <_CopyItems Include="nuget\buildTransitive\**" />
       <_CopyItems Include="@(_ResizetizerFiles)" />
     </ItemGroup>
-    <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_MauiBuildTasksLocation)%(RecursiveDir)%(_CopyItems.Arch)" ContinueOnError="true" Retries="0" />
+    <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_MauiBuildTasksLocation)%(RecursiveDir)%(_CopyItems.Arch)" ContinueOnError="true" Retries="3" RetryDelayMilliseconds="1000" />
     <MakeDir Directories="$(IntermediateOutputPath)adjustments" />
     <AdjustReferencedAssemblyVersion
       Assembly="$(_AdjustmentsAssembly)"
       ReferencedAssembly="$(_AdjustmentsReferencedAssembly)"
       OutputAssembly="$(IntermediateOutputPath)adjustments\Svg.Skia.dll" />
-    <Copy SourceFiles="$(IntermediateOutputPath)adjustments\Svg.Skia.dll" DestinationFiles="$(_MauiBuildTasksLocation)Svg.Skia.dll" ContinueOnError="true" Retries="0" />
+    <Copy SourceFiles="$(IntermediateOutputPath)adjustments\Svg.Skia.dll" DestinationFiles="$(_MauiBuildTasksLocation)Svg.Skia.dll" ContinueOnError="true" Retries="3" RetryDelayMilliseconds="1000" />
     <ItemGroup>
       <FileWrites Include="$(IntermediateOutputPath)adjustments\Svg.Skia.dll" />
     </ItemGroup>

